package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "generator-jhipster-entity-audit",
-  "version": "3.1.1",
+  "version": "4.0.0",
```

Comment:
You should not update the package version. The maintainer will do it when a new package version will be released.